### ISSUE TITLE
Category section is missing a screen reader text

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwenty#412]
The category section does not have a screen reader text, so the context for these links is missing.

**Comment by Patil2099** [comment]
@carolinan Can you help me in solving this issue. I am new to Open-Source?
Thanks!

**Comment by carolinan** [comment]
@Patil2099 I recommend starting here: https://github.com/WordPress/twentytwenty/blob/master/CONTRIBUTING.md

Currently the limited implementation is included here:
https://github.com/WordPress/twentytwenty/blob/master/template-parts/entry-header.php#L23
https://github.com/WordPress/twentytwenty/blob/master/template-parts/content-cover.php#L66

Looking closer, it seems that the theme does include a screen reader text for the categories, but that the code is not used:
https://github.com/WordPress/twentytwenty/blob/master/inc/template-tags.php#L297

A new location needs to be added to twentytwenty_get_post_meta for the categories above the post title, and a design decision is needed here for whether or not the markup and icon should be used.

Because of this I will remove the good first issue label.
